feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -106,19 +114,20 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice, p1 *ondatra.Port, p2 *on
 
 	// Configure network instance
 	t.Logf("*** Configuring network instance on DUT ... ")
-	niConfPath := gnmi.OC().NetworkInstance("10")
-	niConf := configNetworkInstance("10", &ateDst)
-	gnmi.Replace(t, dut, niConfPath.Config(), niConf)
-	niConfPath = gnmi.OC().NetworkInstance("20")
-	niConf = configNetworkInstance("20", &ateDst2)
+	niConfPath := gnmi.OC().NetworkInstance("VRF-10")
+	Intfname := *i2.Name + ".10"
+	niConf := configNetworkInstance("VRF-10", Intfname, "10")
 	gnmi.Replace(t, dut, niConfPath.Config(), niConf)
 
 	// Configure default NI and forwarding policy
 	t.Logf("*** Configuring default instance forwarding policy on DUT ...")
 	dutConfPath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance)
 	gnmi.Replace(t, dut, dutConfPath.Type().Config(), oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
+
 	policyDutConf := configForwardingPolicy()
 	gnmi.Replace(t, dut, dutConfPath.PolicyForwarding().Config(), policyDutConf)
+
+	// CALL static route config
 }
 
 func configInterfaceDUT(i *oc.Interface, me, peer *attrs.Attributes, subintfindex uint32, vlan uint16) *oc.Interface {
```

Comment:
The peer parameter is unused here, too. Can you remove?

Reply:
right. will be removed..

Reply:
Done in new commit.